demos/classification_benchmark_demo/cpp/main.cpp

```diff
@@ -26,90 +26,93 @@
-DEFINE_string(res, "1280x720", image_grid_resolution_message);
-DEFINE_bool(auto_resize, false, input_resizable_message);
-DEFINE_bool(no_show, false, no_show_message);
-DEFINE_uint32(time, std::numeric_limits<gflags::uint32>::max(), execution_time_message);
-DEFINE_string(u, "", utilization_monitors_message);
-
-static void showUsage() {
-    std::cout << std::endl;
-    std::cout << "classification_benchmark_demo [OPTION]" << std::endl;
-    std::cout << "Options:" << std::endl;
-    std::cout << std::endl;
-    std::cout << "    -h                        " << help_message << std::endl;
-    std::cout << "    -i \"<path>\"               " << image_message << std::endl;
-    std::cout << "    -m \"<path>\"               " << model_message << std::endl;
-    std::cout << "    -auto_resize              " << input_resizable_message << std::endl;
-    std::cout << "    -labels \"<path>\"          " << labels_message << std::endl;
-    std::cout << "    -layout \"<string>\"        " << layout_message << std::endl;
-    std::cout << "    -gt \"<path>\"              " << gt_message << std::endl;
-    std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
-    std::cout << "    -nthreads \"<integer>\"     " << num_threads_message << std::endl;
-    std::cout << "    -nstreams \"<integer>\"     " << num_streams_message << std::endl;
-    std::cout << "    -nireq \"<integer>\"        " << num_inf_req_message << std::endl;
-    std::cout << "    -nt \"<integer>\"           " << ntop_message << std::endl;
-    std::cout << "    -res \"<WxH>\"              " << image_grid_resolution_message << std::endl;
-    std::cout << "    -no_show                  " << no_show_message << std::endl;
-    std::cout << "    -time \"<integer>\"         " << execution_time_message << std::endl;
-    std::cout << "    -u                        " << utilization_monitors_message << std::endl;
-}
+namespace{
+constexpr char h_msg[] = "show the help message and exit";
+DEFINE_bool(h, false, h_msg);
+
+constexpr char i_msg[] = "an input to process. The input must be a single image, a folder of images";
+DEFINE_string(i, "", i_msg);
+
+constexpr char labels_msg[] = "path to .txt file with labels";
+DEFINE_string(labels, "", labels_msg);
+
+constexpr char m_msg[] = "path to an .xml file with a trained model";
+DEFINE_string(m, "", m_msg);
+
+constexpr char auto_resize_msg[] = "enables resizable input";
+DEFINE_bool(auto_resize, false, auto_resize_msg);
+
+constexpr char d_msg[] = "specify the target device to infer on. "
+    "The list of available devices is shown below. The demo will look for a suitable plugin for device specified. "
+    "Default value is CPU";
+DEFINE_string(d, "CPU", d_msg);
+
+constexpr char gt_msg[] = "path to ground truth .txt file";
+DEFINE_string(gt, "", gt_msg);
+
+constexpr char layout_msg[] = "specify inputs layouts."
+    "Ex. \"[NCHW]\" or \"input1[NCHW],input2[NC]\" in case of more than one input";
```

Comment:
The format is different in https://github.com/openvinotoolkit/open_model_zoo/blob/272d2d7001cefc712337f9cde2e2ca02b6b3c923/demos/object_detection_demo/cpp/main.cpp#L56
Please recheck this.